### PR TITLE
Add CLI support for the Per API Logging feature

### PR DESCRIPTION
**Pull request by binodmx**
Fixes https://github.com/wso2/product-apim/issues/12098

### CODE REVIEW

File: import-export-cli/cmd/getAPILogging.go [hunk 1]
```diff
@@ -75,9 +75,12 @@ func init() {
 
 	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingAPIId, "api-id", "i",
 		"", "API ID")
+	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingTenantId, "tenant-id", "",
+		"", "TENANT ID")
 	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingEnvironment, "environment", "e",
 		"", "Environment of the APIs which the API loggers should be displayed")
 	getApiLoggingCmd.Flags().StringVarP(&getAPILoggingCmdFormat, "format", "", "", "Pretty-print API loggers "+
 		"using Go Templates. Use \"{{ jsonPretty . }}\" to list all fields")
+	_ = getApiLoggingCmd.MarkFlagRequired("tenant-id")
```
Comment (npamudika) [inline import-export-cli/cmd/getAPILogging.go:84]: Without mandating the tenant-id flag, shall we use carbon.super if the falg is not provided?

Comment (npamudika) [reply import-export-cli/cmd/getAPILogging.go:84]: We can check whether the value is empty and if so set carbon.super as the tenant-id value when calling the rest APIs

File: import-export-cli/cmd/getAPILogging.go [hunk 2]
```diff
@@ -75,9 +75,12 @@ func init() {
 
 	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingAPIId, "api-id", "i",
 		"", "API ID")
+	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingTenantId, "tenant-id", "",
+		"", "TENANT ID")
```
Comment (npamudika) [inline import-export-cli/cmd/getAPILogging.go:79]: Let's use as "Tenant ID"

File: import-export-cli/cmd/setAPILogging.go [hunk 3]
```diff
@@ -74,12 +74,15 @@ func init() {
 	SetCmd.AddCommand(setApiLoggingCmd)
 
 	setApiLoggingCmd.Flags().StringVarP(&setApiLoggingAPIId, "api-id", "i",
-		"", "Api ID")
+		"", "API ID")
+	setApiLoggingCmd.Flags().StringVarP(&setApiLoggingTenantId, "tenant-id", "",
+		"", "TENANT ID")
 	setApiLoggingCmd.Flags().StringVarP(&setApiLoggingLogLevel, "log-level", "",
-		"", "Api ID")
+		"", "Log level")
```
Comment (npamudika) [inline import-export-cli/cmd/setAPILogging.go:81]: Let's use as "Log Level"

File: import-export-cli/cmd/setAPILogging.go [hunk 4]
```diff
@@ -74,12 +74,15 @@ func init() {
 	SetCmd.AddCommand(setApiLoggingCmd)
 
 	setApiLoggingCmd.Flags().StringVarP(&setApiLoggingAPIId, "api-id", "i",
-		"", "Api ID")
+		"", "API ID")
+	setApiLoggingCmd.Flags().StringVarP(&setApiLoggingTenantId, "tenant-id", "",
```
Comment (npamudika) [inline import-export-cli/cmd/setAPILogging.go:78]: Let's use as "Tenant ID"

File: import-export-cli/utils/constants.go [hunk 5]
```diff
@@ -112,7 +112,6 @@ const defaultAPILoggingListEndpoint = "apis?logging-enabled=false"
 const defaultAPILoggingApisEndpoint = "apis"
 
 const DefaultEnvironmentName = "default"
-const DefaultTenantDomain = "carbon.super"
```
Comment (npamudika) [inline import-export-cli/utils/constants.go]: Let's add the const back and use it when setting the default value if the tenant-id flag is empty

File: import-export-cli/cmd/getAPILogging.go (hunk 1 above)
Comment (binodmx) [reply import-export-cli/cmd/getAPILogging.go]: tenant-id is not mandatory now. default value is set to carbon.super

File: import-export-cli/utils/constants.go (hunk 5 above)
Comment (binodmx) [reply import-export-cli/utils/constants.go]: added the constant back.

File: import-export-cli/cmd/getAPILogging.go (hunk 2 above)
Comment (binodmx) [reply import-export-cli/cmd/getAPILogging.go]: done

File: import-export-cli/cmd/setAPILogging.go (hunk 3 above)
Comment (binodmx) [reply import-export-cli/cmd/setAPILogging.go]: done

File: import-export-cli/cmd/setAPILogging.go (hunk 4 above)
Comment (binodmx) [reply import-export-cli/cmd/setAPILogging.go:78]: done

File: import-export-cli/cmd/getAPILogging.go [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+		}
+	}
+}
+
+func init() {
+	GetCmd.AddCommand(getApiLoggingCmd)
+
+	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingAPIId, "api-id", "i",
+		"", "API ID")
+	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingTenantId, "tenant-id", "",
```
Comment (npamudika) [inline import-export-cli/cmd/getAPILogging.go:78]: Shall we use this as "tenant-domain" as we need to pass the domain name here to the flag, and not the ID. Let's correct the wording in all the other places.

Comment (binodmx) [reply import-export-cli/cmd/getAPILogging.go]: corrected

**Pull request merged**